angularjs/angularjs-quiz.md

```diff
@@ -50,151 +50,151 @@ export class OrderService {
-- [ ] this.httpClient.post<Order>(this.orderUrl, order) .subscribe();
+- [ ] this.httpClient.url(this.orderUrl).post(order);
+- [ ] this.httpClient.send(this.orderUrl, order);
+- [ ] this.httpClient.post<Order>(this.orderUrl, order);
+- [x] this.httpClient.post<Order>(this.orderUrl, order).subscribe();
 
-#### Q6. What is the RouterModule.forRoot method used for? What is the null method used for?
-- [ ] registering any providers that you intend to use in routed components registering any providers that you intend to use in routed components
-- [ ] registering route definitions at the root application level registering route definitions at the root application level
-- [ ] indicating that Angular should cheer on your routes to be successful indicating that Angular should cheer on your routes to be successful
-- [ ] declaring that you intend to use routing only at the root level
+#### Q6. What is the RouterModule.forRoot method used for?
+- [ ] Registering any providers that you intend to use in routed components.
+- [x] Registering route definitions at the root application level.
+- [ ] Indicating that Angular should cheer on your routes to be successful.
+- [ ] Declaring that you intend to use routing only at the root level.
 
 #### Q7. Which DOM elements will this component metadata selector match on?
 ```javascript
-    @Component({
+@Component({
     selector: 'app-user-card',
     . . .
-    })
+})
 ```
-- [ ] any element with the attribute app-user-card, such as <div app-user-card></div>any element with the attribute null, such as null
-- [ ] the first instance of <app-user-card></app-user-card>the first instance of null
-- [ ] all instances of <app-user-card></app-user-card>all instances of null
-- [ ] all instances of <user-card></user-card>
+- [ ] Any element with the attribute app-user-card, such as `<div app-user-card></div>`.
+- [ ] The first instance of `<app-user-card></app-user-card>`.
+- [x] All instances of `<app-user-card></app-user-card>`.
+- [ ] All instances of `<user-card></user-card>`.
 
-#### Q8. What is the correct template syntax for using the built-in ngFor structural directive to render out a list of productNames?What is the correct template syntax for using the built-in null structural directive to render out a list of null?
+#### Q8. What is the correct template syntax for using the built-in ngFor structural directive to render out a list of productNames?
 - [ ]
-```javascript
+```html
 <ul>
     <li [ngFor]="let productName of productNames">
     {{ productName }}
     </li>
 </ul>
 ```
 - [ ]
-```javascript
+```html
 <ul>
     <li ngFor="let productName of productNames">
     {{ productName }}
     </li>
 </ul>
 ```
-- [ ]
-```javascript
+- [x]
+```html
 <ul>
     <li *ngFor="let productName of productNames">
     {{ productName }}
     </li>
 </ul>
 ```
 - [ ]
-```javascript
+```html
 <ul>
     <? for productName in productNames { ?>
     <li>{{ productName }}</li>
     <? } ?>
 </ul>
 ```
```

Comment:
To be properly rendered, this lists need the code to be indented (not a standard, this is GitHub's markdown). If you select the code and TAB it a bit, then GitHub is likely to render it nicely.